python/pylibcudf/pylibcudf/gpumemoryview.pyx

```diff
@@ -26,6 +26,26 @@ cdef class gpumemoryview:
         # TODO: Need to respect readonly
         self.ptr = cai["data"][0]
 
+    @staticmethod
+    cdef gpumemoryview from_pointer(Py_ssize_t ptr, object owner):
```

Comment:
Are we sure we want `Py_ssize_t`? Why a signed type specifically? I think I would prefer `from stdint cimport uintptr_t` here.

Reply:
Same question

Reply:
That's a good question. I do want to do this, but let's look at this in a follow-up because changing it would entail changing the member of the gpumemoryview itself and I don't recall if that causes any problems.

Reply:
I opened https://github.com/rapidsai/cudf/issues/18324 to follow up on this.

Reply:
I can take a look at 18324 if you don't get to a first.